common/pkg/models/vllm.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+// VllmModelName returns the vllm model name from the model ID used in LLM Operator.
```

Comment:
nitpick: If we follow the Go convention, the function name should be `VLLMModelName`.